Makefile

```
@@ -53,9 +53,9 @@ space := $(null) $(null)
 WEBOTS_HOME_PATH=$(subst $(space),\ ,$(strip $(subst \,/,$(WEBOTS_HOME))))
 include $(WEBOTS_HOME_PATH)/resources/Makefile.os.include
 
-.PHONY: clean cleanse debug distrib release webots_dependencies webots_target clean-docs docs
+.PHONY: clean cleanse debug distrib release webots_dependencies webots_target clean-docs docs proto_list
 
-release debug profile: docs webots_target
+release debug profile: docs webots_target proto_list
```

We consider the proto list is a dependency of Webots, e.g., Webots needs it to run, we should  move `proto_list` as a `webots_target` dependency on line 104.
```suggestion
release debug profile: docs webots_target
```